<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tower Of Hanoi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>841</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>231</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="VL1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>55</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton1">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton2">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton3">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton4">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>231</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="VL2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>55</number>
      </property>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>231</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="VL3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>55</number>
      </property>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonB">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>330</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Tower B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonC">
     <property name="geometry">
      <rect>
       <x>582</x>
       <y>330</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Tower C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonA">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Tower A</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_Hint">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>781</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose     Tower</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Timer">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>01 : 30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Count">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0     Moves</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Timer">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGuide"/>
   <addaction name="separator"/>
   <addaction name="actionRestart"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionGuide">
   <property name="text">
    <string>How To Play</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
